metric_learn/itml.py

```diff
@@ -87,10 +87,8 @@ def __init__(self, gamma=1., max_iter=1000, convergence_threshold=1e-3,
     self.gamma = gamma
     self.max_iter = max_iter
     self.convergence_threshold = convergence_threshold
-    self.init = init  # explain that it is good to keep the scale with
-    # the bounds
-    # TODO: see for other inits how it behave wrt the bound
-    self.A0 = A0  # TODO: deprecate
+    self.init = init
```

Comment:
@bellet if I remember correctly we were saying that since ITML computes bounds, it is needed that the init is coherent with these bounds, and with the default identity it should be OK, but otherwise it would not, so here are some of the things that could be done (I don't remember if we said other things):
- we could compute the bounds after the init, but maybe if the init is very bad, the bounds would be bad too ? 
- we could rescale the init so that we have the maximum of constraints that satisfy the bounds (this avoids to wait for gradient iterations to converge to this)
What do you think ?